src/theme/foundations/colors.ts

```diff
@@ -129,6 +129,12 @@ export const colors = {
         start: '#fbe7e8',
         end: '#f8d1d4'
       }
+    },
+    priorityScale: {
+      bgGradient: {
+        start: 'rgba(224, 120, 97, 0.1)',
```

Comment:
I don't see that we are using this format in any part of the codebase. I used it bc we are using `rgba` for those values that require `alpha`.

However, not a strong opinion, but since you are talking about consistency, perhaps using `rgba` here is more consistent. What do you think?